src/client/pages/OfferNew/Perils/InsuranceValues/index.tsx

```diff
@@ -88,14 +89,14 @@ export const InsuranceValues: React.FC<Props> = ({ offerQuote }) => {
         {[...offerQuote.insuranceTerms.entries()].map(
           ([insuranceTermType, insuranceTerm]) => {
             return (
-              <Link
+              <TermsLink
                 key={insuranceTermType}
                 href={insuranceTerm.url}
-                target="_blank"
+                external
```

Comment:
Maybe use target instead for a more familiar API 
```suggestion
                target="_blank"
```

Reply:
Yeah I like that better!

Another approach would be to determine it directly form the URL (starts with http...) but perhaps it's better to be explicit.

Reply:
Cool, let's do that! I think we shouldn't default all external links to have `target="_blank"` and only use it for edge cases.